yogstation/code/modules/cargo/cargo_packs.dm

```diff
@@ -51,6 +51,14 @@
 	crate_name = "Advanced Crate Container"
 	crate_type = /obj/structure/closet/crate/large
 
+/datum/supply_pack/misc/goats
+	name = "Goat Crate"
+	desc = "We have recently bought out a company called Goat Tech Industries that have been experimenting with goat dna, since there are still a ton of goats left and we need to get rid of most of these goats we decided to sell some of them to you guys!"
```

Comment:
all other descriptions are actual descriptions, please rewrite this so it's not written like it's talking _to_ the viewer